d2bs/kolbot/libs/common/Misc.js

```diff
@@ -1847,23 +1850,27 @@ MainLoop:
 
 	// Report script errors to logs/ScriptErrorLog.txt
 	errorReport: function (error, script) {
-		var i, h, m, s, date, msg, oogmsg, filemsg, source, stack,
+		var i, y, mo, d, h, m, s, date, dateString, msg, oogmsg, filemsg, source, stack,
 			stackLog = "";
 
 		date = new Date();
+		y = date.getFullYear();
+		mo = date.getMonth() + 1;
+		d = date.getDate();
 		h = date.getHours();
 		m = date.getMinutes();
 		s = date.getSeconds();
+		dateString = "[" + y + "/" + (d < 10 ? "0" + d : d) + "/" + (mo < 10 ? "0" + mo : mo) + " " + (h < 10 ? "0" + h : h) + ":" + (m < 10 ? "0" + m : m) + ":" + (s < 10 ? "0" + s : s) + "]";
```

Comment:
try this for date string:
```
var datestring = "[" + ("0" + d.getDate()).slice(-2) + "/" + ("0"+(d.getMonth()+1)).slice(-2) + "/" + d.getFullYear() + " " + ("0" + d.getHours()).slice(-2) + ":" + ("0" + d.getMinutes()).slice(-2) + ("0" + d.getSeconds()).slice(-2) + "]";
```

notation should be either day / month / year or year / month / date
year / day / month seems arbitrary

Reply:
i meant to do year month date but probably messed up while moving stuff around.
uploaded fix.